etl/data_helpers/geo_old.py

```diff
@@ -1,3 +1,8 @@
+"""
```

Comment:
I'd suggest removing this module if it's not used anywhere. It can be confusing and someone may start using it by mistake.

Reply:
Idem with `test_geo_old`.